### PR TITLE
Rdiscrowd 3933 add logger change errormsg

### PR DESCRIPTION
**Pull request by pwills11**
*Issue number of the reported bug or feature request: #<number>*

**Describe your changes**
A clear and concise description of the changes you have made.

**Testing performed**
Describe the testing you have performed to ensure that the bug has been addressed, or that the new feature works as planned.

**Additional context**
Add any other context about your contribution here.

### CODE REVIEW

File: pybossa/view/projects.py [hunk 1]
```diff
@@ -3094,9 +3094,12 @@ def sync_project(short_name):
                 'A request error occurred while syncing {}: {}'
                 .format(project.short_name, str(res.__dict__)))
             msg = gettext(
```
Comment (ywrsusan) [inline pybossa/view/projects.py:3096]: Considering adding 
`current_app.logger.info('response code: {}'.format(str(res.status_code))`

Comment (pwills11) [reply pybossa/view/projects.py:3096]: Okay, I will add this. Do you think that i should add logger.info at all instances which already contain logger.error (such as the one seen here?)

Comment (ywrsusan) [reply pybossa/view/projects.py:3096]: Yea that would be great

**Review by kbecker42 — COMMENTED** [review]
Looks good. Just need to fix the merge conflict before committing. Thank you.

**Review by kbecker42 — DISMISSED** [review]
Consider adding unit test coverage after investigating with new log lines. Thank you.

**Review by ciaracian — APPROVED** [review]
Ok it sounds like this is good to go even with the loss in test coverage.

File: test/test_syncer/test_project_syncer.py [hunk 2]
```diff
@@ -195,3 +195,28 @@ def test_sync_api_integration(self, mock_cat):
...
+    @patch('pybossa.syncer.category_syncer.CategorySyncer.get_target', return_value={'id': 1})
+    @patch('pybossa.syncer.project_syncer.ProjectSyncer.cache_target')
+    @patch('pybossa.syncer.project_syncer.ProjectSyncer._sync')
+    @patch('pybossa.syncer.requests')
+    def test_sync_update_existing(self, fake_requests, mock_update, mock_cache, mock_cat, mock_get):
+        project_syncer = ProjectSyncer(self.target_url, self.target_key)
+        user = UserFactory.create(admin=True, email_addr=u'user@test.com')
+        project_syncer.syncer = user
+
+	      res = Response()
```
Comment (ywrsusan) [inline test/test_syncer/test_project_syncer.py:210]: Looks like this indentation needs to be fix.
You can run tests on you local by following [here](https://bbgithub.dev.bloomberg.com/dtwebservices/gigwork-docker-dev#run-test-in-docker-and-debug-using-pdb)

File: test/test_syncer/test_project_syncer.py [hunk 3]
```diff
@@ -195,3 +195,27 @@ def test_sync_api_integration(self, mock_cat):
...
+    @patch('pybossa.syncer.project_syncer.ProjectSyncer.cache_target')
+    @patch('pybossa.syncer.project_syncer.ProjectSyncer._sync')
+    @patch('pybossa.syncer.requests')
+    def test_sync_update_existing(self, fake_requests, mock_update, mock_cache, mock_cat, mock_get):
+        project_syncer = ProjectSyncer(self.target_url, self.target_key)
+        user = UserFactory.create(admin=True, email_addr=u'user@test.com')
+        project_syncer.syncer = user
+
+        res = Response()
+        res.ok=False
```
Comment (ywrsusan) [inline test/test_syncer/test_project_syncer.py:211]: Should be `res._ok` and `res._content`, see [here](https://github.com/bloomberg/pybossa/blob/07aae932e7fa9e2bdadda0d9f2613022906d93bd/test/test_syncer/test_project_syncer.py#L33)

**Review by ywrsusan — APPROVED** [review]

**Pull request merged**